native/src/main/java/io/ballerina/lib/sikulix/SikulixScreen.java

```diff
@@ -0,0 +1,334 @@
+            return location.getY();
+        } catch (Exception e) {
+            return Utils.getBError(e.getMessage(), e);
+        }
+    }
+
+    public static Object getX(BObject bScreen) {
+        try {
+            return getScreen(bScreen).getX();
+        } catch (Exception e) {
+            return Utils.getBError(e.getMessage(), e);
+        }
+    }
+
+    public static Object getY(BObject bScreen) {
+        try {
+            return getScreen(bScreen).getY();
+        } catch (Exception e) {
+            return Utils.getBError(e.getMessage(), e);
+        }
+    }
+
+    public static Object getW(BObject bScreen) {
+        try {
+            return getScreen(bScreen).getW();
+        } catch (Exception e) {
+            return Utils.getBError(e.getMessage(), e);
+        }
+    }
+
+    public static Object getH(BObject bScreen) {
+        try {
+            return getScreen(bScreen).getH();
+        } catch (Exception e) {
+            return Utils.getBError(e.getMessage(), e);
+        }
+    }
+
+    public static Object find(BObject bScreen, BString imagePath) {
+        try {
+            Match match = getScreen(bScreen).find(imagePath.toString());
+            return getMatchBObject(match);
+        } catch (Exception e) {
+            return Utils.getBError(e.getMessage(), e);
+        }
+    }
+
+    public static Object findText(BObject bScreen, BString text) {
+        try {
+            Match match = getScreen(bScreen).findText(text.toString());
+            return getMatchBObject(match);
+        } catch (Exception e) {
+            return Utils.getBError(e.getMessage(), e);
+        }
+    }
+
+    public static Object findAll(BObject bScreen, BString imagePath) {
+        try {
+            Iterator<Match> matches = getScreen(bScreen).findAll(imagePath.toString());
+            return ValueCreator.createArrayValue(getMatchBObjectArr(matches), getMatchArrayType());
+        } catch (Exception e) {
+            return Utils.getBError(e.getMessage(), e);
+        }
+    }
+
+    private static BObject[] getMatchBObjectArr(Iterator<Match> matches) {
+        List<Match> matchList = new ArrayList<>();
+        while (matches.hasNext()) {
+            matchList.add(matches.next());
+        }
+        BObject[] matchArr = new BObject[matchList.size()];
+        for (int i = 0; i < matchList.size(); i++) {
+            matchArr[i] = getMatchBObject(matchList.get(i));
+        }
+        return matchArr;
+    }
+
+    public static ArrayType getMatchArrayType() {
+        Type type = TypeCreator.createObjectType(MATCH_OBJECT_TYPE, ModuleUtils.getModule(), 0);
+        return TypeCreator.createArrayType(type);
+    }
+
+    private static Screen getScreen(BObject bScreen) {
+        return (Screen) bScreen.getNativeData(SCREEN_OBJECT);
+    }
+
+    private static BObject getMatchBObject(Match match) {
+        BObject matchBObj = ValueCreator.createObjectValue(ModuleUtils.getModule(), MATCH_OBJECT_TYPE, (Object) null);
```

Comment:
```suggestion
        BObject matchBObj = ValueCreator.createObjectValue(ModuleUtils.getModule(), MATCH_OBJECT_TYPE, null);
```
Do we need to cast this here ?

Reply:
When removing the casting, getting below warning message during the gradle build.

```
C:\Users\Dharshi\Documents\lib\module-sikulix\native\src\main\java\io\ballerina\lib\sikulix\SikulixMatch.java:232: warning: non-varargs call of varargs method with inexact argument type for last parameter;
        BObject matchBObj = ValueCreator.createObjectValue(ModuleUtils.getModule(), MATCH_OBJECT_TYPE, null);
                                                                                                     
  ^
  cast to Object for a varargs call
  cast to Object[] for a non-varargs call and to suppress this warning
1 warning
```

When running an example below error occurs: 

```
error: Cannot read the array length because "fieldValues" is null
```